tables/TaQL/TaQLNodeDer.cc

```diff
@@ -1613,22 +1631,25 @@ TaQLCreTabNodeRep* TaQLCreTabNodeRep::restore (AipsIO& aio)
 {
   TaQLMultiNode with = TaQLNode::restoreMultiNode (aio);
   TaQLNode giving = TaQLNode::restoreNode (aio);
+  TaQLMultiNode likeDrop = TaQLNode::restoreMultiNode (aio);
   TaQLMultiNode columns = TaQLNode::restoreMultiNode (aio);
   TaQLNode limit = TaQLNode::restoreNode (aio);
   TaQLMultiNode dminfo = TaQLNode::restoreMultiNode (aio);
-  TaQLCreTabNodeRep* node = new TaQLCreTabNodeRep (with, giving, columns,
-                                                   limit, dminfo);
+  TaQLCreTabNodeRep* node = new TaQLCreTabNodeRep (with, giving, likeDrop,
+                                                   columns, limit, dminfo);
```

Comment:
Use std::unique_ptr where needed.
Note that result of restore function is put in a TaQLNose object which is exception-safe.

Reply:
Even when they don't throw now they may throw in the feature. I'm not fond of using raw owning pointers in C++11, since they tend to lead to bugs. Since casacore originated before C++11 it doesn't use the new smart pointers a lot. I still think it would be good to look at using them more. Especially in code that doesn't affect the API.